src/PhpSlang/Collection/Generic/Component/PartitionableCollection.php

```diff
@@ -110,6 +111,8 @@
      * @param Collection $pairs
      *
      * @return Closure
+     *
+     * @SuppressWarnings(PHPMD.UnusedLocalVariable)
```

Comment:
we use second argument and 1st one is not used :/ so I don't see any way not to define it. If it'd be scala I could simply use placeholder syntax instead in this place (but it's not scala, so I can't)